clang/lib/CodeGen/Targets/PPC.cpp

```diff
@@ -421,6 +503,11 @@ ABIArgInfo PPC32_SVR4_ABIInfo::classifyReturnType(QualType RetTy) const {
     }
   }
 
+  bool IsComplexInRegABI =
+      getCodeGenOpts().getComplexInRegABI() == CodeGenOptions::CMPLX_InGPR;
+  if (IsComplexInRegABI && RetTy->isAnyComplexType())
```

Comment:
you do not need to introduce a new variable `IsComplexInRegABI` here since it only be used once.

change to 
`if ((getCodeGenOpts().getComplexInRegABI() == CodeGenOptions::CMPLX_InGPR) && RetTy->isAnyComplexType())`